docs/source/backends/zerotier.rst

```diff
@@ -274,30 +274,32 @@ Client specific settings
+| key name               | type    | default                    | description                                                                                               |
++========================+=========+============================+===========================================================================================================+
+| ``name``               | string  |  ``ow_zt``                 | name of the zerotier network                                                                              |
++------------------------+---------+----------------------------+-----------------------------------------------------------------------------------------------------------+
+| ``networks``           | list    | ``[{}]``                   | list of dictionaries containing strings with **16-digit** hexadecimal network IDs for joining,            |
+|                        |         |                            |                                                                                                           |
+|                        |         |                            | along with a corresponding custom **10-digit** ZeroTier interface name for each network                   |
+|                        |         |                            |                                                                                                           |
+|                        |         |                            | **note:** ensure that the list includes at least one such dictionary                                      |
++------------------------+---------+----------------------------+-----------------------------------------------------------------------------------------------------------+
+| ``config_path``        | string  | ``/etc/openwisp/zerotier`` | path to the persistent configuration directory                                                            |
++------------------------+---------+----------------------------+-----------------------------------------------------------------------------------------------------------+
+| ``copy_config_path``   | string  | ``'1'``                    | specifies whether to copy the configuration file to RAM                                                   |
+|                        |         |                            |                                                                                                           |
+|                        |         |                            | ``'0'`` - No, ``'1'`` - Yes, this prevents writing to flash in zerotier controller mode                   |
++------------------------+---------+----------------------------+-----------------------------------------------------------------------------------------------------------+
+| ``secret``             | string  | ``{{secret}}``             | identity secret of the zerotier client (network member), leave it default to be automatically determined  |
```

Comment:
Configuration variables are not a part of netjsonconfig, but openwisp-controller. I would set empty string here and point out the default zerotier behaviour, i.e. it would generate automatically. 

It is okay to use it in the code, but in documentation we shall give examples such that netjsonconfig can be used independently from openwisp-controller.